notebooks-auth/test_thredds.ipynb

```diff
@@ -56,15 +56,15 @@
     "import requests\n",
     "print(\"Setup configuration parameters...\")\n",
     "\n",
-    "PAVICS_HOST = os.getenv(\"PAVICS_HOST\", \"pavics.ouranos.ca\").rstrip(\"/\")\n",
+    "PAVICS_HOST = (os.getenv(\"PAVICS_HOST\") or \"pavics.ouranos.ca\").rstrip(\"/\")\n",
```

Comment:
Just curious, `os.getenv("VAR_NAME", default_value)` do not work so you have to do `os.getenv("VAR_NAME") or default_value`?  I know both achieve the same effect so I am not against the other format but I just wonder what is the subtleties that make you change to the other form.

Reply:
`os.getenv("VAR_NAME", default_value)` will return an empty string or None if the variable is defined with `VAR_NAME=` or `VAR_NAME=""`.  In this case, I want to consider empty equivalent as undefined and revert to the default.

Reply:
I see!  That "empty value" problem did happen via Jenkins build request or you are being defensive for when invoking `pytest` manually directly on the command-line and user can accidentally introduce the problem?